src/components/Field.tsx

```diff
@@ -0,0 +1,95 @@
+import { 
+	SyntheticEvent,
+	useState,
+	createContext, 
+	useContext,
+	HTMLAttributes,
+} from "react";
+import { pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/lib/Either";
+import * as A from "fp-ts/lib/Array";
+
+import styles from 'styles/field.module.scss';
+import { ErrorsList } from "helpers/validators";
+import { logPipe } from "helpers/functional";
+
+type InputValue = string | boolean | null;
+type ValidatorProps = { errors: E.Either<string[], string> };
+export type ValidatorType = { [key: string]: () => E.Either<string, string> }[];
+
+type Props = {
+	fieldId: string,
+	label: string,
+	isRequired?: boolean,
+} & HTMLAttributes<HTMLInputElement>
+
+export const FormContext = createContext<ValidatorType>([]);
+
+export default function Field({ fieldId, label, isRequired = false, ...props }: Props) {
+	const validators = useContext<any>(FormContext);
```

Comment:
`any` type is completely forbidden in our apps

Reply:
Fixed in [caed15a](https://github.com/rafaelpradella/pradella-fp-page/pull/2/commits/caed15a45cfd8eb5581b6dec633557ff9797822b) Moved to PR #2 since it caused a lot of conflicts.